### PR TITLE
bump to 0.2.0

### PR DESCRIPTION
**Pull request by lucascolley**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by lucascolley** [comment]
@conda-forge-admin, please rerender

**Comment by lucascolley** [comment]
@conda-forge-admin, please rerender

**Comment by lucascolley** [comment]
@conda-forge-admin please pre-render

**Comment by lucascolley** [comment]
@conda-forge-admin, please rerender

**Comment by lucascolley** [comment]
hey @conda-forge/help-python, does anybody know why I am seeing a SHA mismatch in CI?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+{% set python_min = "3.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/array_api_extra-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/array_api_extra-{{ version }}.tar.gz
   sha256: 91b685fc90b6c5364760053127b9ee3d256caeb3563ca327c6fc98123fd5c973
```
Comment (ocefpaf) [inline recipe/meta.yaml]:
```suggestion
  sha256: 9b7719795a823ae29f4735adc32ad820c1d975c0317b86b9d614580d420b3cda
```

New file, new checksum. You can get it from PyPI , compute it locally, or wait for the bot to send the PR for you with the version and checksum update.
Screenshot from 2024-11-20 13-34-19

Comment (lucascolley) [reply recipe/meta.yaml]: thanks!

> or wait for the bot to send the PR for you with the version and checksum update

Does that mean that this version bump PR would have been created automatically if I didn't make one first?

Comment (ocefpaf) [reply recipe/meta.yaml]: It should. It usually takes the bot 1-2 hours after the release, but you can force it [with the bot command](https://conda-forge.org/docs/maintainer/infrastructure/#conda-forge-admin-please-update-version). If nothing happens, it is likely a bug somewhere and the [status page](https://conda-forge.org/status/#version) usually has the error message for debugging.

**Pull request merged**